lib/ansible/module_utils/common/collections.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c), Sviatoslav Sydorenko <ssydoren@redhat.com> 2018
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+"""Collection of low-level utility functions."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+from ..six import binary_type, text_type
+from ._collections_compat import Sequence
```

Comment:
Import tests will pass with:

```python
from ansible.module_utils.six import binary_type, text_type
from ansible.module_utils.common._collections_compat import Sequence
```

Reply:
I don't know if we want to start using relative imports -- we aren't using them anywhere else yet. If we do want to start using them the import test will need to be updated. In the meantime we should avoid relative imports.

Reply:
What if I go and fix the test?

Reply:
We need to have a team discussion before making such changes.

Reply:
Also, there are other areas within ansible-test which will require updates to support relative imports. Import analysis for change classification is one of them (this is separate from the import sanity test).